packages/fxa-settings/src/models/integrations/oauth-integration.ts

```diff
@@ -311,6 +315,11 @@ export class OAuthIntegration extends BaseIntegration<OAuthIntegrationFeatures>
   }
 
   async getPermissions() {
+    // TODO: Not 100% sure about this...
+    if (!this.data.scope) {
```

Comment:
On the confirm password reset page, using the any of the oauth password reset functional tests, scope is not in the URL. So without this fix, the code would fail... I could definitely use a sanity check on whether or not this expected or not.

Here's a link to the tests that were failing prior to changing this. https://app.circleci.com/pipelines/github/mozilla/fxa/45015/workflows/c04699ce-9f1f-4a27-95ff-77c3646792dc/jobs/465766

Reply:
> scope is not in the URL

I don't understand this part.  Scopes are not expected to be in the URL.

Reply:
> @chenba Hmm isn't it? If I go to Firefox Monitor prod and click "sign in", I see `scope` in the URL. Isn't this required for oauth flows?
> 

Oh I was definitely wrong.  Sorry.  I don't know what I had in mind at that time.  😬